make/harbor.cfg

```diff
@@ -133,14 +133,21 @@ clair_db_username = postgres
 
 #Clair default database
 clair_db = postgres
-
-
 ################### end of HA section #####################
+
 #************************END INITIAL PROPERTIES************************
+
 #The following attributes only need to be set when auth mode is uaa_auth
 uaa_endpoint = uaa.mydomain.org
 uaa_clientid = id
 uaa_clientsecret = secret
 uaa_verify_cert = true
 uaa_ca_cert = /path/to/ca.pem
-#############
+
+
+### Docker Registry setting ###
+#registry_storage_provider can be: filesystem, s3, gcs, azure, etc.
+registry_storage_provider = filesystem
+#registry_storage_provider_cfg is a comma separated "key: value" pairs, e.g. "key1: value, key2: value2".
+#Refer to https://docs.docker.com/registry/configuration/#storage for all available configuration.
+registry_storage_provider_cfg =
```

Comment:
Just my two cents: 
what about we provide a storage configuration file , the content of the file will be same as https://docs.docker.com/registry/configuration/#storage  says.  so that we don't need to explain the format. 
eg:
registry_storge_conf = storage.cfg

Reply:
got your point. then we need to parse or read yaml formatted file storage.cfg and replace its content with the placeholder in config.yml.
It's doable. And if more config sections are needed (besides the storage section), then more separate cfg snippet file are needed. BTW, we also need to explain the yaml format and note the indent in the snippet storage.cfg.

Reply:
After discussion with Yixing, we will ask the user a question to confirm shared storage is configured for Registry:
  Is the Harbor Docker Registry configured to use shared storage (e.g. NFS, S3, GCS, etc.)? [yes/no]: